core/src/test/scala/com/broadinstitute/dsp/DBTestHelper.scala

```diff
@@ -58,54 +58,52 @@ object DBTestHelper {
     implicit xa: HikariTransactor[IO]
   ): IO[Long] =
     sql"""INSERT INTO CLUSTER
-         (clusterName, 
-         googleProject, 
-         operationName, 
-         status, 
-         hostIp, 
-         createdDate, 
-         destroyedDate, 
-         initBucket, 
-         creator, 
-         serviceAccount, 
-         stagingBucket, 
-         dateAccessed, 
-         autopauseThreshold, 
-         defaultClientId, 
-         stopAfterCreation, 
-         kernelFoundBusyDate, 
-         welderEnabled, 
-         internalId, 
-         runtimeConfigId, 
+         (clusterName,
```

Comment:
a bit hard to see from the diff..but what's changed in this list?

Reply:
Sorry about that, I removed `stopAfterCreation` and it's corresponding value which were causing the unit tests to fail since this field had been removed from the leo API (in a previous ticket not related to this)